### ISSUE TITLE
multiple "Text to style did not contain any tokens. Returning empty string." warnigns

### ISSUE THREAD

**Issue by IndrajeetPatil** [r-lib/styler#460]
I am getting a lot of `warnings` when I do `styler:::style_pkg()` on my R [package](https://github.com/IndrajeetPatil/ggstatsplot) and I am not sure how to get rid of them or even why they are being produced in the first place. Any ideas?

![image](https://user-images.githubusercontent.com/11330453/51423750-9eadd380-1bea-11e9-9247-4657934be693.png)

I don't how to create a reprex for this, but here is the session information-
``` r
sessioninfo::session_info()
#> - Session info ----------------------------------------------------------
#>  setting  value                                             
#>  version  R Under development (unstable) (2018-11-30 r75724)
#>  os       Windows 10 x64                                    
#>  system   x86_64, mingw32                                   
#>  ui       RTerm                                             
#>  language (EN)                                              
#>  collate  English_United States.1252                        
#>  ctype    English_United States.1252                        
#>  tz       Asia/Calcutta                                     
#>  date     2019-01-19                                        
#> 
#> - Packages --------------------------------------------------------------
#>  package     * version    date       lib source                    
#>  assertthat    0.2.0      2017-04-11 [1] CRAN (R 3.5.1)            
#>  cli           1.0.1.9000 2018-10-30 [1] Github (r-lib/cli@56538e3)
#>  crayon        1.3.4      2017-09-16 [1] CRAN (R 3.5.1)            
#>  digest        0.6.18     2018-10-10 [1] CRAN (R 3.5.1)            
#>  evaluate      0.12       2018-10-09 [1] CRAN (R 3.5.1)            
#>  highr         0.7        2018-06-09 [1] CRAN (R 3.5.1)            
#>  htmltools     0.3.6      2017-04-28 [1] CRAN (R 3.5.1)            
#>  knitr         1.21       2018-12-10 [1] CRAN (R 3.6.0)            
#>  magrittr      1.5        2014-11-22 [1] CRAN (R 3.5.1)            
#>  Rcpp          1.0.0      2018-11-07 [1] CRAN (R 3.6.0)            
#>  rmarkdown     1.11       2018-12-08 [1] CRAN (R 3.6.0)            
#>  sessioninfo   1.1.1      2018-11-05 [1] CRAN (R 3.6.0)            
#>  stringi       1.2.4      2018-07-20 [1] CRAN (R 3.6.0)            
#>  stringr       1.3.1      2018-05-10 [1] CRAN (R 3.5.1)            
#>  withr         2.1.2      2018-03-15 [1] CRAN (R 3.5.1)            
#>  xfun          0.4        2018-10-23 [1] CRAN (R 3.6.0)            
#>  yaml          2.2.0      2018-07-25 [1] CRAN (R 3.5.1)            
#> 
#> [1] C:/Users/inp099/Documents/R/win-library/3.6
#> [2] C:/Program Files/R/R-devel/library
```

<sup>Created on 2019-01-19 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by lorenzwalthert** [comment]
Thanks. First, it's an unproblematic warning that should probably turned into a message. Which version of styler are you running? This potentially comes from `.Rmd` code chunks that were empty or roxygen code examples that were empty. What was the exact code you were running? I.e. just `styler::style_pkg()` with all default values?

**Comment by IndrajeetPatil** [comment]
I am using `styler`'s GitHub version with the latest commit (d1f3df9) included.  

Yes, I do just `styler::style_pkg()` with all default values.

**Comment by lorenzwalthert** [comment]
I just pushed a fix with #462 and I re-styled your package successfully without any warning. Can you confirm?

**Comment by IndrajeetPatil** [comment]
Thanks! I can confirm that this works as expected.
